src/lpython/semantics/python_attribute_eval.h

```diff
@@ -74,6 +75,16 @@ struct AttributeHandler {
         return ASR::make_IntegerBitLen_t(al, loc, s, int_type, nullptr);
     }
 
+    static ASR::asr_t* eval_int_to_bytes(ASR::expr_t *s, Allocator &al, const Location &loc,
+            Vec<ASR::expr_t*> &args, diag::Diagnostics &/*diag*/) {
+        if (args.size() != 2) {
+            throw SemanticError("int.to_bytes(i, s) takes exactly two arguments", loc);
+        }
+        ASR::ttype_t *char_type = ASRUtils::TYPE(ASR::make_Character_t(al, loc,
+                                        1, -1, nullptr, nullptr, 0));
+        return ASR::make_IntegerToBytes_t(al, loc, s, char_type, nullptr);
```

Comment:
The code from `python_ast_to_asr` that computes the compile time evaluation should be moved here and it would be returned in the last argument.